backend/src/main/resources/application-azure-dev.yaml

```diff
@@ -5,6 +5,9 @@ simple-report:
     upload-enabled: true
     upload-url: "https://staging.prime.cdc.gov/api/reports?option=SkipInvalidItems" # same destination as test
     upload-schedule: "0 30 * * * *" # hourly on the :30 to differentiate from test, which sends on the :00
+  azure-reporting-queue:
+    enabled: true
+    name: test-event-publishing
```

Comment:
Note that these are scoped/namespaced w/in the Storage Accounts, so it's fine that these all have the same name